src/spatial/models/report_datafetcher.py

```
@@ -0,0 +1,142 @@
+import requests
+import torch 
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from huggingface_hub import login
+from configure import Config
+
+# Initialize gemini
+import google.generativeai as genai
+from configure import Config
```

Ruff

Redefinition of unused `Config` from line 5

Remove definition: `Config`

(F811)